Merge pull request #2 from vijapandey/master

vijapandey opened the pull request:
updated files by upload

vijapandey committed:
Merge pull request #2 from vijapandey/master

updated files by upload

vijapandey committed:
Add files via upload

Add some core Spring , and core java stuff [enum, class loader, checked/unchecked exception, memory leak]"

ghost commented:
## DeepCode's analysis on [#e8f80e](https://www.deepcode.ai/app/gh/vijapandey/javatesting/61815aa4bbeff96db1450f121c48f4a36959d047/vijapandey/javatesting/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=3&) found:

- :information_source: **3** minor issues. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">To provide random values, the java.util.Random object must be reused and not created for every call. Occurrences: <ul><li><a href="https://github.com/vijapandey/javatesting/blob/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/src/main/java/com/javatesting/simple/LeakTest.java#L21">LeakTest.java:21</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/vijapandey/javatesting/61815aa4bbeff96db1450f121c48f4a36959d047/vijapandey/javatesting/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/pr/_/%2Fsrc%2Fmain%2Fjava%2Fcom%2Fjavatesting%2Fsimple%2FLeakTest.java/java%2Fdc_interfile_project%2FReuseRandom%2Ftest/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">Either rethrow this java.lang.InterruptedException or set the interrupted flag on the current thread with 'Thread.currentThread().interrupt()'. Otherwise the information that the current thread was interrupted will be lost. Occurrences: <ul><li><a href="https://github.com/vijapandey/javatesting/blob/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/src/main/java/com/javatesting/simple/LeakTest.java#L33">LeakTest.java:33</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/vijapandey/javatesting/61815aa4bbeff96db1450f121c48f4a36959d047/vijapandey/javatesting/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/pr/_/%2Fsrc%2Fmain%2Fjava%2Fcom%2Fjavatesting%2Fsimple%2FLeakTest.java/java%2Fdc_interfile_project%2FcatchingInterruptedExceptionWithoutInterrupt%2Ftest/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
    <tr>
      <td width="77%">System.out.println is used. Occurrences: <ul><li><a href="https://github.com/vijapandey/javatesting/blob/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/src/main/java/com/javatesting/collection/LinkedHashMapExample.java#L36">LinkedHashMapExample.java:36</a></li> <li><a href="https://github.com/vijapandey/javatesting/blob/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/src/main/java/com/javatesting/collection/LinkedHashMapExample.java#L37">LinkedHashMapExample.java:37</a></li> <li><a href="https://github.com/vijapandey/javatesting/blob/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/src/main/java/com/javatesting/collection/LinkedHashMapExample.java#L48">LinkedHashMapExample.java:48</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/vijapandey/javatesting/61815aa4bbeff96db1450f121c48f4a36959d047/vijapandey/javatesting/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/pr/_/%2Fsrc%2Fmain%2Fjava%2Fcom%2Fjavatesting%2Fcollection%2FLinkedHashMapExample.java/java%2Fpmd%2FSystemPrintln/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/vijapandey/javatesting/61815aa4bbeff96db1450f121c48f4a36959d047/vijapandey/javatesting/e8f80eb4a18e3dddd01433b7347f9de2cb6a9e4c/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=3&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=vijapandey)

vijapandey committed:
Add files via upload

Add some core Spring , and core java stuff [enum, class loader, checked/unchecked exception, memory leak]"

vijapandey committed:
Add files via upload

Add some core Spring , and core java stuff [enum, class loader, checked/unchecked exception, memory leak]"

vijapandey committed:
Add files via upload

Add some core Spring , and core java stuff [enum, class loader, checked/unchecked exception, memory leak]"

vijapandey committed:
Add files via upload

Add some core Spring , and core java stuff [enum, class loader, checked/unchecked exception, memory leak]"

vijapandey committed:
modification and adding java file

vijapandey committed:
added

vijapandey committed:
commit

vijapandey committed:
ignore file

vijapandey committed:
Added ignore file

vijapandey committed:
Stream java

vijapandey committed:
latest commit

vijapandey committed:
pom properties
